org.moonsoft.MoonPlayer.json

```diff
@@ -0,0 +1,195 @@
+{
+  "app-id": "org.moonsoft.MoonPlayer",
```

Comment:
Incorrect app-id, you don't control moonsoft.org.

Reply:
OK, I will open a new branch to rename it.

Reply:
Don't need a new branch (if you mean Flathub branch).